lib/components/ms-textarea/MsTextarea.vue

```diff
@@ -0,0 +1,136 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="input-container">
+    <span
+      id="label"
+      class="form-label"
+      v-show="label"
+    >
+      {{ $msTranslate(label) }}
+    </span>
+
+    <ion-textarea
+      class="textarea"
+      :class="{
+        input: !$props.disabled,
+      }"
+      ref="textareaRef"
+      :placeholder="$msTranslate(placeholder)"
+      :value="modelValue"
+      @ion-input="onChange($event.detail.value || '')"
+      :disabled="$props.disabled"
+      v-bind="$attrs"
+      :auto-grow="autogrow || false"
+      :minlength="minlength || DEFAULT_MINLENGTH"
+      :maxlength="maxlength || DEFAULT_MAXLENGTH"
+      :rows="rows || 10"
+    />
+    <span
+      v-show="errorMessage !== ''"
+      class="form-error form-helperText"
+    >
+      <ion-icon
+        class="form-error-icon"
+        :icon="warning"
+      />
+      {{ $msTranslate(errorMessage) }}
+    </span>
+    <span
+      class="count"
+      v-show="showCount || true"
+    >
+      {{ modelValue?.length }} / {{ maxlength || DEFAULT_MAXLENGTH }}
```

Comment:
What happens if you set `:showCount="true"` without `v-model`?

Reply:
`/ maxlength` displays.